net/libfab/libfab_internal.h

```diff
@@ -48,7 +48,7 @@ extern struct m0_net_xprt m0_net_libfab_xprt;
 
 #define LIBFAB_ADDR_LEN_MAX	INET6_ADDRSTRLEN
 #define LIBFAB_PORT_LEN_MAX	6
-#define LIBFAB_ADDR_STRLEN_MAX  (LIBFAB_ADDR_LEN_MAX + LIBFAB_PORT_LEN_MAX + 1)
+#define LIBFAB_ADDR_STRLEN_MAX  (LIBFAB_ADDR_LEN_MAX + LIBFAB_PORT_LEN_MAX + 32)
```

Comment:
Why is this change needed?

Reply:
From #1119:

```
net/libfab/libfab.c:735:22: error: ':12345:' directive writing 7 bytes into a region of size between 3 and 50 [-Werror=format-overflow=]
   sprintf(buf, "%s@%s:12345:%d:%d",
                      ^~~~~~~
```

`LIBFAB_ADDR_STRLEN_MAX` is used as the buffer length. It's obvious that `+ 1` in the old code is not enough to accommodate what is tried to be snprintf-ed here besides the `LIBFAB_ADDR_LEN_MAX` and `LIBFAB_PORT_LEN_MAX`. Increasing its size fixed it.

Reply:
I think this is required when we try to LNET format EP address in libfabric.

Thank you.